config/sync/views.view.search_db.yml

```diff
@@ -116,9 +116,9 @@ display:
       exposed_form:
         type: bef
         options:
-          submit_button: Search
+          submit_button: Søg
           reset_button: true
-          reset_button_label: Reset
+          reset_button_label: Nustil
```

Comment:
Typo: Nustil -> Nulstil

Reply:
Fixed in https://github.com/itk-dev/hoeringsportal/pull/472/commits/2f5fe9c7577dbcee8198535904a355454c443e44